Sources/swift-test/test.swift

```diff
@@ -31,7 +32,7 @@ func test(path: String, xctestArg: String? = nil) throws -> Bool {
     }
 #endif
 
-    let result: Void? = try? system(args)
+    let result: Void? = try? system(args, environment: NSProcessInfo.processInfo().environment)
```

Comment:
Can we get a comment here to clarify the intent, so it is obvious why we pass the full env?